lib/run.js

```diff
@@ -32,10 +32,14 @@ function validateOpts(options) {
     
     validateNumberProp(options, 'rate');
     validateNumberProp(options, 'concurrent');
-    
-    // hack to enable concurrency mode,
-    // since forkmaster expects this to be defined
-    options.rate = options.rate || 0;
```

Comment:
Dunno if it works in all cases, but it worked when I ran concurrent and rate tests. I guess you fixed it?